_posts/2011-01-02-app-self-service.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
In the nav bar this article shows up before "how to Submit an apk through Developer portal". Probably should be the other way around.